packages/client/src/components/SafeSettings/components/Owners/AddSafeOwner.js

```diff
@@ -10,14 +10,24 @@ const AddSafeOwner = ({ onCancel, onNext, safeOwners }) => {
   const [name, setName] = useState("");
   const [address, setAddress] = useState("");
   const [addressValid, setAddressValid] = useState(false);
+  const [addressBorderClass, setAddressBorderClass] = useState("border-light");
```

Comment:
Can we implement this with the "input is-success" class so we can apply the same style on all inputs? https://bulma.io/documentation/form/input/